src/aiperf/metrics/types/completed_request_count_metric.py

```
@@ -0,0 +1,32 @@
+    """
+
+    tag = "completed_request_count"
+    header = "Completed Requests (Success + Error)"
+    short_header = "Completed"
+    short_header_hide_unit = True
+    unit = GenericMetricUnit.REQUESTS
+    display_order = 1075
+    flags = MetricFlags.NO_INDIVIDUAL_RECORDS
+    required_metrics = {
+        RequestCountMetric.tag,
+    }
```

Ruff (0.15.10)

[warning] Mutable default value for class attribute

(RUF012)